internal/channel/channel.go

```diff
@@ -20,109 +19,149 @@ type Channel struct {
 	Config string `db:"config" json:"-"` // excluded from JSON config dump as this may contain sensitive information
 
 	Logger *zap.SugaredLogger
```

Comment:
`Logger` is no longer accessed externally, so it can be changed to `logger` now.